src/run.ts

```diff
@@ -4,12 +4,13 @@ import * as path from 'path';
 import * as toolCache from '@actions/tool-cache';
 
 import { downloadKubectl, getStableKubectlVersion } from "./utilities/kubectl-util";
-import { getExecutableExtension, isEqual } from "./utilities/utility";
+import { getDeploymentConfig, getExecutableExtension, isEqual, isValidAction } from "./utilities/utility";
 
 import { Kubectl } from './kubectl-object-model';
 import { deploy } from './utilities/strategy-helpers/deployment-helper';
 import { promote } from './actions/promote';
 import { reject } from './actions/reject';
+import * as AzureTraceabilityHelper from "./utilities/../traceability/azure-traceability-helper";

```

Comment:
Looks like the path can directly begin with `./traceability`

Reply:
Changed.